rosbag2/include/rosbag2/readers/sequential_reader.hpp

```diff
@@ -29,18 +29,21 @@
 #include "rosbag2/serialization_format_converter_factory_interface.hpp"
 #include "rosbag2/visibility_control.hpp"
 
-
 namespace rosbag2
 {
+namespace readers
+{
 
-class ROSBAG2_LOCAL SequentialReaderImpl : public BaseReaderInterface
+class ROSBAG2_PUBLIC SequentialReader : public BaseReaderInterface
```

Comment:
Is sequential reader meant to be instantiated outside the library? It would be preferable to keep headers private so that we can keep updating rosbag2 w/o breaking ABI compat.